packages/types/src/mpe-temporal-to-server.ts

```diff
@@ -0,0 +1,18 @@
+import * as z from 'zod';
+import { MpeWorkflowState } from './mpe';
+
+export const MpeRejectAddingTracksRequestBody = z.object({
+    roomID: z.string().uuid(),
+    deviceID: z.string().uuid(),
+});
+export type MpeRejectAddingTracksRequestBody = z.infer<
+    typeof MpeRejectAddingTracksRequestBody
+>;
+
+export const MpeAcknowledgeAddingTracksRequestBody = z.object({
+    state: MpeWorkflowState,
+    deviceID: z.string().uuid(),
```

Comment:
This is not a big deal, but I think that a deviceID should always be associed to a userID
As far as for logging, keeping trace is concerned, if the device just got evicted